### PR TITLE
Update osx build doc for namecoin

### PR DESCRIPTION
**Pull request by whitj00**

### CODE REVIEW

File: doc/build-osx.md [hunk 1]
```diff
@@ -20,17 +20,17 @@ Dependencies
...
+Build Namecoin Core
 ------------------------
 
-1. Clone the bitcoin source code and cd into `bitcoin`
+1. Clone the namecoin source code and cd into `namecoin`
 
-        git clone https://github.com/bitcoin/bitcoin
-        cd bitcoin
+        git clone https://github.com/namecoin/namecoin-core
+        cd namecoin
```
Comment (JeremyRand) [inline doc/build-osx.md]: I think this should be `namecoin-core`, not `namecoin`.

File: doc/build-osx.md [hunk 2]
```diff
@@ -49,37 +49,37 @@ Build Bitcoin Core
...
-    tail -f $HOME/Library/Application\ Support/Bitcoin/debug.log
+    tail -f $HOME/Library/Application\ Support/Namecoin/debug.log
 
 Other commands:
 -------
 
-    ./src/bitcoind -daemon # Starts the bitcoin daemon.
-    ./src/bitcoin-cli --help # Outputs a list of command-line options.
-    ./src/bitcoin-cli help # Outputs a list of RPC commands when the daemon is running.
+    ./src/namecoin -daemon # Starts the namecoin daemon.
```
Comment (JeremyRand) [inline doc/build-osx.md]: I think this should be `namecoind`, not `namecoin`.

**Comment by JeremyRand** [comment]
Hey Whit!  Great to see you here.  Apologies for the delay getting this reviewed, I've got a bunch of things on my plate this week.  I've noted a couple of things that might be errors.  I don't have easy access to an OS X machine to test this more thoroughly -- pinging @cassiniNMC since he can test these instructions much more easily than I can.

Ping me if Cassini hasn't gotten back to you in a week.  :)

**Comment by JeremyRand** [comment]
The people from this issue: https://github.com/namecoin/namecoin-core/issues/83 might be interested in this PR, and could possibly test the instructions.

**Comment by JeremyRand** [comment]
@whitj00 Any luck fixing the 2 things I noted?  (If they're errors.)

File: doc/build-osx.md (hunk 1 above)
Comment (khaledJabr) [inline doc/build-osx.md]: @JeremyRand 
Tested it on Mac OS 10.11.16 and confirm that `namecoin-core` is the right one.

**Comment by khaledJabr** [comment]
Tested that the pull request  f0717c23d049a31fcc4330d86cb97356c9a540ca  works with the two changes proposed by @JeremyRand  in comments and confirm it works.

**Comment by JeremyRand** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Leaving this closed since #154 fixes the issue.

-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJZDm17AAoJELPy0WV4bWVwIyQP/iNOaDMRwbZL/orsuGkbXrRJ
VFcDSMSVJ5uTxQEfTJXBeF3uS0L5mtfDMUzNke4ewspNHrFcMWnQ31tMaWnwmpOg
Odul4binIATZrEczzyMMYU5gPx/6lAnhot9iE70GGnwdl4I5il+MQaLvmT6YHEF4
WbXGMd7OQ4iou6qmTjcait7/olm6dRLs3+sGj/6uMCs8OKEQBeaaMsc3wFZhUl6N
qSA/nkT73gHV1V8EkKXKRv7evTS0mRzs40O3AMDncB8CmjjJsXmqrtic4hYCFxYL
m2+2o2/mddcV+PLzokGYzWXmiJmlKcjOxM4FcxFkRDR+ekKtvb4u6pjys5vnO3xt
+/TX3MJXO1tE0kJhqnO5gHD4BMO0l8dA5uU7pooAtTAyKgjkdZfYKiBEM4WkgEAH
+JsN+qxqcDprunV7ma+GKVn00VLP5C+wcp2uGZZKhIvcM4y3YKOOantDBH1y8t+E
FIVxuYZm1O4aNbMS2oL0KugsugL8O+smOWe5VOeJhX1pd0FhsM+bD7m8n2KHvN84
y/mwSGhfXGUt51JJtTJALDahOws2uGCdIEqK/7V8kEMmRWMsWd8OIOoLE3sk2JS9
9jVRpNMg3HqoTtcfz1t++pN0p45M+MiXMvre61YolVMNPIkl3xvKH6LQLrlHOrfD
I49J49Ql+MBKgWfWsPaB
=SLrD
-----END PGP SIGNATURE-----

**Pull request closed without merging**